### DISCUSSION TITLE
CSS- one bundle for all styles or distributed among modules?

### DISCUSSION THREAD

**Discussion by meniYud** [frehner/modern-guide-to-packaging-js-library#11] [Q&A] [answered]
What do you do with your stylesheets? When you bundle your library code (and keep file structure), Do you create one big 'bundle.css' (the 'simpler' way in most bundlers) ? Or is it better to keep each css next to its consumer js file? (i.e. If the end user consumes only 'button' from my library, it makes no scence for them to include my entire bundle.css ...)

**Comment by frehner** [comment]
Great question! I'll have to do some research on this one; if you also happen to find sources please post them here as well. 👍

**Comment by frehner** [comment, marked as answer]
https://github.com/frehner/modern-guide-to-packaging-js-library#split-out-your-css-files
